app/metrics.yaml

```diff
@@ -205,6 +205,131 @@ events:
       - fenix-core@mozilla.com
     expires: "2020-09-01"
 
+onboarding:
+  fxa_auto_signin:
+    type: event
+    description:
+      The onboarding automatic sign in card was tapped.
+    bugs:
+      - https://github.com/mozilla-mobile/fenix/issues/10824
+    data_reviews:
+      - https://github.com/mozilla-mobile/fenix/pull/11867
+    notification_emails:
+      - fenix-core@mozilla.com
+      - erichards@mozilla.com
+    expires: "2020-09-01"
+  fxa_manual_signin:
+    type: event
+    description:
+      The onboarding manual sign in card was tapped.
+    bugs:
+      - https://github.com/mozilla-mobile/fenix/issues/10824
+    data_reviews:
+      - https://github.com/mozilla-mobile/fenix/pull/11867
+    notification_emails:
+      - fenix-core@mozilla.com
+      - erichards@mozilla.com
+    expires: "2020-09-01"
+  privacy_notice:
+    type: event
+    description:
+      The onboarding privacy notice card was tapped.
+    bugs:
+      - https://github.com/mozilla-mobile/fenix/issues/10824
+    data_reviews:
+      - https://github.com/mozilla-mobile/fenix/pull/11867
+    notification_emails:
+      - fenix-core@mozilla.com
+      - erichards@mozilla.com
+    expires: "2020-09-01"
+  pref_toggled_private_browsing:
+    type: event
+    description:
+      The private browsing preference was selected from the onboarding card.
```

Comment:
Does this automatically trigger the PB mode being saved to the metrics ping for private browsing setting? https://github.com/mozilla-mobile/fenix/blob/master/docs/metrics.md#metrics-1

Reply:
Yes it does!